middlewares/cors_test.go

```diff
@@ -12,20 +14,20 @@ import (
 func TestCorsMiddleware(t *testing.T) {
 
 	router := there.NewRouter()
-	router.Use(Cors(AllowAllConfiguration()))
+	router.Use(Cors(CorsAllowAllConfiguration()))
 	router.Get("/", func(request there.Request) there.Response {
-		return there.Status(there.StatusOK)
+		return there.Status(status.OK)
 	})
 
-	request := httptest.NewRequest(there.MethodGet, "/", nil)
+	request := httptest.NewRequest(string(there.MethodGet), "/", nil)
```

Comment:
`MethodGet` already defined as a string, not sure if need to cast.

Reply:
Forgot to remove it. Before Method was its own type, which required a cast.